### PR TITLE
add 'quiet' formatter as child of simple_text formatter.

### PR DESCRIPTION
**Pull request by dhempy**
Let's try this again...single commit, tests passing, changeme comment in correct spot.

### CODE REVIEW

**Comment by bbatsov** [comment]
That's better, you'll still have to rebase this on top of the current `master` branch. You should also have a look at our commit message guidelines in CONTRIBUTING.md.

File: lib/rubocop/formatter/quiet_formatter.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+
+module RuboCop
+  module Formatter
+    # If no offences are found, no output is displayed.
```
Comment (jonas054) [inline lib/rubocop/formatter/quiet_formatter.rb]: We use the American spelling "offenses" throughout the project.

**Comment by dhempy** [comment]
Thanks for the feedback, Bozhidar.  Do you mean I should merge my feature
branch back into my master, and then pull request from there? And that
should be a single commit, I assume?

-dave

**Comment by dhempy** [comment]
OK…I think I’ve got it all sorted.  Look for a new PR in a moment. 

Thanks for your patience,
-dave

**Pull request closed without merging**